nss-ice-phantoms/BangazonSite
# Photo upload and display on Product Details

# Description

Can now upload photos when creating a product.  Photos will be displayed on Product Details page.  This does not include the functionality to:

1) Show a thumbnail in product list views
2) Remove a photo or add a new photo in the Product Edit view

## Type of change

# Testing Instructions

1.  git fetch --all
2.  git checkout rr-photos
3.  Run BangazonSite
4. Create a new product and add a photo.  Go to the product details and verify that the photo shows.

# Checklist:
